status-im/status-legacy
# Add new welcome page after generating their key

fixes #8135 
(related with #9054 )

### Summary

Add new welcome page after they generate their key.

### Review notes
![Screenshot_1572545235](https://user-images.githubusercontent.com/3349973/67975003-b674b680-fc56-11e9-99ed-c4c3e4ee61b0.png)

![Screenshot_1572545262](https://user-images.githubusercontent.com/3349973/67975021-bc6a9780-fc56-11e9-910a-a4aab571ab11.png)



### Testing notes
Tested on Nexus5X, Pixel2,3

##### Functional

- welcome-view

### Steps to test
- Open Status (first user)
- Generate Key
- Choose a key and name
- Select key storage
- Create a password
- Confirm your password
- Enable fingerprint or pass
- Enable notifications or pass
- new welcome page is appeared
- Click "Let's go" button, it shows 'Empty state chat' right away.



status: wip
